sdk/identity/identity/package.json

```diff
@@ -122,7 +122,6 @@
     "@types/uuid": "^8.0.0",
     "@types/chai": "^4.1.6",
     "chai": "^4.2.0",
-    "assert": "^1.4.1",
```

Comment:
Is this the only change needed? I was expecting changes in the tests 🤔

Reply:
No changes needed in tests (at least for Identity and other packages), they are passing.

Reply:
Awesome!